bundle/manifests/cert-manager-operator.clusterserviceversion.yaml

```diff
@@ -219,7 +219,7 @@ metadata:
     features.operators.openshift.io/token-auth-aws: "true"
     features.operators.openshift.io/token-auth-azure: "true"
     features.operators.openshift.io/token-auth-gcp: "true"
-    olm.skipRange: '>=1.13.0 <1.14.0'
+    olm.skipRange: '>=1.13.1 <1.14.0'
```

Comment:
Would it block users on version 1.13.0 wanting to upgrade to 1.14.0?

Reply:
Users will have to upgrade to 1.13.1 before they can upgrade to 1.14.0 (this is in accordance with upgrade paths we've rolled out in the past) i.e. 1.13.0 -> 1.13.1 -> 1.14.0. For users installing the operator with Automatic approval mode, this will be done automatically seamlessly.